docs/src/main/sphinx/admin/resource-groups.rst

```diff
@@ -10,16 +10,65 @@ it does not cause running queries to fail; instead new queries become queued.
 A resource group may have sub-groups or may accept queries, but may not do both.
 
 The resource groups and associated selection rules are configured by a manager, which is pluggable.
-Add an ``etc/resource-groups.properties`` file with the following contents to enable
-the built-in manager that reads a JSON config file:
+
+You can use a file-based or a database-based resource group manager:
+
+* Add a file ``etc/resource-groups.properties``
+* Set the ``resource-groups.configuration-manager`` property to ``file`` or ``db``
+* Add further configuration properties for the desired manager.
+
+File resource group manager
+---------------------------
+
+The file resource group manager reads a JSON configuration file, specified with
+``resource-groups.config-file``:
 
 .. code-block:: text
 
     resource-groups.configuration-manager=file
     resource-groups.config-file=etc/resource-groups.json
 
-Change the value of ``resource-groups.config-file`` to point to a JSON config file,
-which can be an absolute path, or a path relative to the Trino data directory.
+The path to the JSON file can be an absolute path, or a path relative to the Trino
+data directory. The JSON file only needs to be present on the coordinator.
+
+Database resource group manager
+-------------------------------
+
+The database resource group manager loads the configuration from a relational database. The
+supported databases are MySQL, PostgreSQL, and Oracle.
+
+.. code-block:: text
+
+    resource-groups.configuration-manager=db
+    resource-groups.config-db-url=jdbc:mysql://localhost:3306/resource_groups
+    resource-groups.config-db-user=username
+    resource-groups.config-db-password=password
+
+If any of the tables required for the database resource group manager do not exist when Trino
+starts, they will be created automatically when Trino starts.
+
```

Comment:
Let's add the following two points too (which are different from file-based setting)

The rules in ``selectors`` table are processed in the descending order of the values in ``priority`` field. 

"resource_groups" table also contains an "environment" field which is matched using on the node.environment property (add ref to https://trino.io/docs/current/installation/deployment.html#node-properties). This allows the  resource group configuration for different Trino clusters to be stored in the same database if required.

Reply:
Can you point me to the code where the `priority` field is used for this purpose? I see `SelectorRecord` has a `getPriority` method but I don't see it used anywhere but I could easily be missing it!

Reply:
ah never mind, I see the SQL query for `getSelectors` has an `ORDER BY priority DESC` sorts the selectors by priority. Thanks!!